SECURITY.md

```diff
@@ -12,4 +12,4 @@ The following versions are currently being supported with security updates.
 
 ## Reporting a Vulnerability
 
-Please email Kyle Mathews @ mathews.kyle@gmail.com
+Please email security@gasbyjs.com
```

Comment:
@janson Is the misspelling on purpose? 😅

Reply:
Nope—that's what I get for copy-paste without checking. 🤦‍♂️